src/olympia/amo/mail.py

```diff
@@ -17,21 +17,31 @@ class DevEmailBackend(BaseEmailBackend):
     `settings.SEND_REAL_EMAIL` is disabled.
 
     BUT even if `settings.SEND_REAL_EMAIL` is disabled, if the targeted
-    email address is in the `settings.EMAIL_QA_ALLOW_LIST` list,
-    the email will be sent.
+    email address is in `dev_email` group users, the email will be sent.
     """
 
+    force_send_mail_group = 'Force Send Mail'
+
+    def __force_send_emails(self):
+        try:
+            dev_email_group = Group.objects.get(name=self.force_send_mail_group)
+            return dev_email_group.users.values_list('email', flat=True)
+        except Group.DoesNotExist:
+            return []
```

Comment:
Simpler version that does a single query (need to import `UserProfile`):
```suggestion
        return UserProfile.objects.filter(
            groups__name=self.force_send_mail_group
        ).values_list('email', flat=True)
```

Reply:
Done.